src/models/game.ts

```diff
@@ -105,8 +111,13 @@ class Game extends Model<InferAttributes<Game>, InferCreationAttributes<Game>> {
   }
 
   // Get the playerNumber of the GamePlayer whose turn it is
-  async turnGamePlayerNumber(): Promise<number> {
-    return (await this.turnGamePlayer()).playerNumber;
+  async turnGamePlayerNumber(): Promise<0 | 1> {
```

Comment:
Added this typing because I didn't know if it was `1 | 2` or `0 | 1`.  I think this adds value as a signature, even though it requires a weird implementation.

Reply:
The server uses whatever the client provides for player numbers. The client is currently providing this input to the createGame mutation:
```
        input: {
          players: players.map((p, i) => {
            return { minaPublicKey: p, name: '', playerNumber: i }
          }),
          arenaHeight: 550,
          arenaWidth: 650
        }
```
where it looks like `i` is index and thus starts at 0.